Readme.md

```diff
@@ -0,0 +1,11 @@
+# Playback Speed Calculator
+
+This calculator will help to know the time it will take to watch the complete video at any speed.
+
+# Video Demo
+
+## Demo
+
+<video width="1400" height="650" controls>
+  <source src="./Demo.mp4" type="video/mp4">

```

Comment:
Hi @Anurag-05-prog , video is not playing because this source file does not exist in this commit. Make sure you add the video in your commit and create a PR

Reply:
Oh, sure. I'll do it.